lib_nbgl/include/nbgl_use_case.h

```diff
@@ -150,6 +150,18 @@ typedef struct {
     nbgl_callback_t callback;  ///< function to call when action button is touched in Home page
 } nbgl_homeAction_t;
 
+/**
+ * @brief The necessary parameters to build a tip-box in first review page and
+ * the modal if this tip box is touched
+ *
+ */
+typedef struct {
+    const char                *text;  ///< text of the tip-box
+    const nbgl_icon_details_t *icon;  ///< icon of the tip-box
+    const char *modalTitle;  ///< title given to modal window displayed when tip-box is touched
+    const nbgl_contentInfoList_t infos;  ///< infos pairs displayed in modal.
```

Comment:
Could we already extend this, for example to allow displaying something like _Center Text_, to just give more informatio, display address, accound name.... ?
It could be just using a union, to be completed when needed?


typedef struct {
    const char                *text;  ///< text of the tip-box
    const nbgl_icon_details_t *icon;  ///< icon of the tip-box
    const char *modalTitle;  ///< title given to modal window displayed when tip-box is touched
    union {
        const nbgl_contentInfoList_t infos;  ///< infos pairs displayed in modal.
    } content;
} nbgl_tipBox_t;

Reply:
Indeed, I will rework it that way, with an additionnal field to indicate the type used in the union